### PR TITLE
Add pip check

### PR DESCRIPTION
**Pull request by jan-janssen**
Checklist
* [ ] Used a [personal fork of the feedstock to propose changes](https://conda-forge.org/docs/maintainer/updating_pkgs.html#forking-and-pull-requests)
* [ ] Bumped the build number (if the version is unchanged)
* [ ] Reset the build number to `0` (if the version changed)
* [ ] [Re-rendered]( https://conda-forge.org/docs/maintainer/updating_pkgs.html#rerendering-feedstocks ) with the latest `conda-smithy` (Use the phrase <code>@<space/>conda-forge-admin, please rerender</code> in a comment in this PR for automated rerendering)
* [ ] Ensured the license file is being packaged.

### CODE REVIEW

**Comment by jan-janssen** [comment]
Waiting for https://github.com/conda-forge/staged-recipes/pull/22992

File: recipe/meta.yaml [hunk 1]
```diff
@@ -29,10 +29,15 @@ requirements:
     - pyzmq
     - regex
     - setproctitle
+    - radical.gtod
```
Comment (mtitov) [inline recipe/meta.yaml]: hi @jan-janssen, is there a reason you want to add `radical.gtod` here? It is vice versa - `radical.gtod` depends on `radical.utils`

Comment (jan-janssen) [reply recipe/meta.yaml]: Somehow I thought it was in the requirements, but you are right it is not.

**Comment by jan-janssen** [comment]
@mtitov Can you take another look?

**Review by mtitov — APPROVED** [review]
Thank you! LGTM!

**Comment by jan-janssen** [comment]
@mtitov I do not have the permissions to merge the pull request, can you merge it? Or are we waiting for @andre-merzky ?

**Comment by mtitov** [comment]
right! apologies, merging now

**Pull request merged**